src/mgxs.cpp

```diff
@@ -380,16 +383,17 @@ Mgxs::Mgxs(const std::string& in_name, const std::vector<double>& mat_kTs,
     // If we are doing nearest temperature interpolation, then we don't need
     // to do the 2nd temperature
     int num_interp_points = 2;
+    std::vector<double> interp(micros.size());
```

Comment:
Can you move this ``std::vector`` initialization and the next to below the ``if (settings::temperature_method == TemperatureMethod::Nearest)...`` line? The comment block above is describing the two lines that set ``num_interp_points`` so its better to have them together and next to that comment.

Reply:
ok